.travis.yml

```diff
@@ -51,7 +51,7 @@ matrix:
     - env:
         - NAME="packaging check"
       install: true
-      script: MAVEN_OPTS='-Xmx3000m' mvn -DskipTests -Dforbiddenapis.skip=true -Dcheckstyle.skip=true -Dpmd.skip=true -Dmaven.javadoc.skip=true -pl '!benchmarks' -B --fail-at-end clean install -Pdist
+      script: MAVEN_OPTS='-Xmx3000m' mvn -DskipTests -Dforbiddenapis.skip=true -Dcheckstyle.skip=true -Dpmd.skip=true -Dmaven.javadoc.skip=true -pl '!benchmarks' -B --fail-at-end clean install -Pdist -Pbundle-contrib-exts
```

Comment:
shoudl we be bundling contrib ext by default?

Reply:
Probably not, just need something to caught this kind of error (my last typo surprisingly passed the CI). I’m still figuring my way around here, should I do it other way?

Reply:
This seems like it is probably the best way to test this since it's just a test to make sure the `dist` profile works correctly and all the extensions are setup correctly to work with pull-deps.